django/contrib/admin/static/admin/js/admin/DateTimeShortcuts.js

```diff
@@ -125,8 +127,7 @@
 
             quickElement(
                 'span', clock_link, '',
-                'class', 'clock-icon',
-                'title', gettext('Choose a Time')
+                'class', 'clock-icon'
```

Comment:
Note that if possible, only include changes that are relevant for the ticket 
Other fixes should be included with other associated tickets (in a separate commit) :+1: